src/trellis/app/entry.py

```
@@ -216,6 +225,66 @@ def _is_pyodide() -> bool:
+        # Import watch module lazily to avoid loading watchfiles in browser
+        from trellis.bundler.watch import watch_and_rebuild
+        from trellis.platforms.common.handler_registry import get_global_registry
+        from trellis.platforms.common.messages import ReloadMessage
+
+        def on_rebuild() -> None:
+            """Broadcast reload message to all connected clients."""
+            registry = get_global_registry()
+            if len(registry) > 0:
+                # Schedule broadcast on the watch thread's event loop
+                # This is safe because we're already in the watch thread
+                asyncio.create_task(registry.broadcast(ReloadMessage()))
```

Ruff (0.14.10)

Store a reference to the return value of `asyncio.create_task`

(RUF006)